### ISSUE TITLE
Octant support

### ISSUE THREAD

**Issue by VarLad** [raphamorim/rio#814]
Are there any plans to support Octants from Unicode 16 in Rio?
More information on them here:
https://github.com/microsoft/cascadia-code/issues/711#issue-2094002225

Chafa supports them now https://github.com/hpjansson/chafa/issues/201
Foot Terminal is getting support for them as well :)

**Comment by raphamorim** [comment]
Of course! I was actually looking into it this week. Haven't started anything yet but it's on my wishlist.
